websocketOutportDriver/client/wsConnClient.go

```diff
@@ -0,0 +1,55 @@
+package client
+
+import (
+	"github.com/gorilla/websocket"
+	"github.com/multiversx/mx-chain-core-go/core/check"
+)
+
+type wsConnClient struct {
+	conn *websocket.Conn
+}
+
+// NewWSConnClient creates a new wrapper over a websocket connection
+func NewWSConnClient() *wsConnClient {
```

Comment:
:+1: for creating this wrapper.

Reply:
Thanks. 
It is very much needed for easier testing